packages/hash/api/src/graph/util.ts

```diff
@@ -160,37 +160,31 @@ export const generateSystemPropertyTypeSchema = (
  */
 export const propertyTypeInitializer = (
   params: PropertyTypeCreatorParams,
-): ((graphApi: GraphApi) => Promise<PropertyTypeWithMetadata>) => {
+): ((context: ImpureGraphContext) => Promise<PropertyTypeWithMetadata>) => {
   let propertyType: PropertyTypeWithMetadata;
 
-  return async (graphApi?: GraphApi) => {
+  return async (ctx?: ImpureGraphContext) => {
```

Comment:
This is why I really don't like the shadowing / re-define lint 🙃  

We're presumably breaking our own conventions about not shortening variable names just so that the lint doesn't complain and these two things don't conflict. Which sort of worries me even more because it'd be too easy to say `context.` inside here and not know there's a problem.

Reply:
Actually maybe we're not even doing that here? The `context` above should just be in the scope of the return type presumably, and not cause problems.

Reply:
It's really annoying that the names clash. I think we don't use the outermost context, so perhaps we can just call it `_` outside and `context` inside?

Reply:
It seems we aren't using those outer contexts at all. Should we get rid of them?

Reply:
197773ac120481acea936543ecee2034e9ee1257

Reply:
<insert egg emoji here>